bin/find_v8_git_versions.py

```diff
@@ -1,4 +1,13 @@
 #!/usr/bin/env python2.7
+"""
+This script exists because depot_tools (the git wrapper used by V8) does not
+fetch fixed versions of the dependencies it downloads via Git. There are some
+20+ git repos fetched. This script extracts the git version hashes from an
+existing V8 source directory (with dependencies pre-fetched) and generates a
```

Comment:
I found the bit after this a little confusing. How about something like "generates a shell script which checks out those specific versions. This can then be called in the future to ensure that later builds always use precisely the same versions as when the shell script was generated."?